upgrade to nativebase 0.5.22

mrphu3074 opened the pull request.

wcandillon commented:
@mrphu3074 doesn't remove the point of using types? Also shouldn't change happen to https://github.com/GeekyAnts/NativeBase/blob/master/index.d.ts now?

mrphu3074 commented:
@wcandillon no, I don't. 
Below is update:

- Updated component's properties follow NativeBase 0.5.22
- Added `<Fab/>`
- Avoid throwing errors when pass not defined properties. By add this snipet beblow interface
```js
/**
* For any props comming soon
*/
[key: string]: any;
```
Ex:
```jsx
<Button active>Label</Button>
// Or
<Tabs>
  <View tabLabel="Tab 1" />
</Tabs>
```
